superset-frontend/package.json

```diff
@@ -145,6 +145,7 @@
     "js-levenshtein": "^1.1.6",
     "js-yaml-loader": "^1.2.2",
     "json-bigint": "^1.0.0",
+    "json-bigint-native": "^1.2.0",
```

Comment:
If `json-bigint` is no longer maintained, can we remove that dep (on line 147)?

Reply:
the line [43](https://github.com/apache/superset/blob/2a852477943f588bfff6ac9dd0452bb9981bb93a/superset-frontend/packages/superset-ui-core/package.json#L43) also need to be removed.

Reply:
will follow up with this